python-avd/pyavd/avd_schema_tools.py

```diff
@@ -40,20 +41,23 @@ def convert_data(self, data: dict) -> list[AvdDeprecationWarning]:
         Returns:
             List of AvdDeprecationWarnings
         """
+        from .vendor.errors import AvdConversionWarning, AvdDeprecationWarning  # pylint: disable=import-outside-toplevel
```

Comment:
could we make some mechanism "ala TYPE_CHECKING" maybe to tell what to import at the top of the file if set in `pyavd/__init__.py` ?

Reply:
Not really, because if this function is called we need to import this.
Note I only moved the vendor imports down, so it will go away as we migrate to native pyavd.